test/lib/DutchDecayLib.t.sol

```diff
@@ -74,27 +74,26 @@ contract DutchDecayLibTest is Test {
         assertEq(DutchDecayLib.decay(2 ether, 1 ether, 100, 200), 1 ether);
     }
 
-    function testDutchDecayBounded(uint256 startAmount, uint256 endAmount, uint256 decayStartTime, uint256 decayEndTime)
+    function testDutchDecayBounded(int256 startAmount, int256 endAmount, uint256 decayStartTime, uint256 decayEndTime)
```

Comment:
why change types to int256 to cast to uint256 in the test?

Reply:
it helps bound the values to a range to prevent overflow.. I tried `vm.assume(startAmount < 2* 255 - 1)` but forge testing would fail saying it hit the max number of failing input.

Reply:
Ah I see, and bound would reject too many probably